docs/refget-design.md

```diff
@@ -0,0 +1,83 @@
+# Design notes: Modos x Refget
+
+
+## Goal
+Use a [refget](http://samtools.github.io/hts-specs/refget.html) api in modos to handle reference sequences in an unambigous and standardized way.
+
+## Considerations
+1. GA4GH published a [Refget Sequence Collections](https://ga4gh.github.io/refget/seqcol/) specification (v1 approved 2025) to:
+    *   allow retrival of sequence collections (usually __reference
+ genomes__)
+    *   specify __sequence collection hashes__, that are composed from the __sequence hashes__ and __sequence attributes__ (names, length,?)(see [Seqcol standard](#seqcol-standard))
```

Comment:
sequence collections would nicely map to the ReferenceGenome concept we have, but I suggest we ignore it for now to keep it simple